roles/rsyslog/tasks/outputs/relp/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Set and check certs and key for relp output
+  include_tasks:
+    file: "{{ role_path }}/tasks/set_certs.yml"
+  vars:
+    __rsyslog_cert_subject: "{{ rsyslog_output_relp }}"
+
+# Deploy configuration files
+- name: Install/Update relp logging packages and generate configuration files in /etc/rsyslog.d
+  vars:
+    __rsyslog_packages: "{{ __rsyslog_output_relp_packages }}"
+    __rsyslog_rules: "{{ __rsyslog_output_relp_rules }}"
+  include_tasks:
+    file: "{{ role_path }}/tasks/deploy.yml"
+
+- name: Create relp output configuration files in /etc/rsyslog.d
+  vars:
+    __rsyslog_packages: []
+    __rsyslog_rules:
+      - name: "output-relp-{{ item.name }}"
+        type: "output"
+        weight: "31"
+        state: "{{ item.state | d('present') }}"
+        sections:
+          - options: "{{ lookup('template', 'output_relp.j2') }}"
+  include_tasks:
+    file: "{{ role_path }}/tasks/deploy.yml"
+  loop: '{{ rsyslog_output_relp }}'
```

Comment:
should be consistent with use of single quotes vs. double quotes - I think you should use double quotes except where you have to use single quotes within a double quote